src/agent/driver-test-runner.js

```diff
@@ -175,6 +175,8 @@ export class DriverTestRunner {
         commandsOutput.push({
           command: command.id,
           output: spokenOutput.join('\n'),
+          // v2 property only
+          presentationNumber: command.presentationNumber,
```

Comment:
Why this one? Just in case it's useful in the future?

Reply:
Yeah, at first it was because I thought it was this one that mattered, but never removed it.  It's not used or transmitted currently so I could remove

Reply:
91c0f1d